src/scenarios/shared/androidhelper.py

```diff
@@ -202,7 +187,7 @@ def setup_device(self, packagename: str, packagepath: str, animationsdisabled: b
 
         testRun = RunCommand(self.startappcommand, verbose=True)
         testRun.run()
-        testRunStats = re.findall(runSplitRegex, testRun.stdout) # Split results saving value (List: Starting, Status, LaunchState, Activity, TotalTime, WaitTime) 
+        testRunStats = re.findall(run_split_regex, testRun.stdout) # Split results saving value (List: Starting, Status, LaunchState, Activity, TotalTime, WaitTime) 
```

Comment:
nit: testRunStat -> test_run_stats

Reply:
Yea, there are many places that need to be updated to match this convention so I will do a new PR that includes many formatting changes.